src/components/header.tsx

```diff
@@ -77,46 +104,59 @@ type HeaderProps = {
    * Search component
    */
   search?: ReactNode;
+  /**
+   * Secondary items
+   */
+  secondaryItems?: Array<HeaderPossibleItem | HeaderDropdown>;
+  /**
+   * Subtext
+   */
+  subtext?: ReactNode;
```

Comment:
Actually realised there is an issue if providing `subtext` and no `secondaryItems`. Will update tomorrow

Reply:
Done